P2CODE/AA1/SenseHat/ADOPTERS.yaml

```diff
@@ -0,0 +1,10 @@
+description: This is a compilation list of the current adopters of the data model SenseHat of the Subject dataModel.Device.  All fields are non mandatory. More info at https://smart-data-models.github.io/data-models/templates/dataModel/CURRENT_ADOPTERS.yaml
```

Comment:
remove the sentence after All fields ...

Reply:
Do you mean to delete this sentence? 
More info at https://smart-data-models.github.io/data-models/templates/dataModel/CURRENT_ADOPTERS.yaml

I still see it in the official data models tho. Please correct me if I am wrong.